CARBALL_VERSION

```diff
@@ -1 +1 @@
-24
+28
```

Comment:
why

Reply:
So it matches the next version number on pypi